misc/connection-limits/CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## 0.2.0 
+## 0.2.1 - unreleased
```

Comment:
```suggestion
## 0.2.1
```

Adding ` - unreleased` is the way to go. Thank you. I will cut a release right away, thus removing it already.